terraform/deployments/sentry/locals.tf

```diff
@@ -0,0 +1,54 @@
+    "licensify"                = ["govuk-developers", "govuk-licensing-team", "replatforming"],
+    "link-checker-api"         = ["govuk-developers", "govuk-publishing", "govuk-publishing-platform", "replatforming"],
+    "local-links-manager"      = ["govuk-content-interactions-on-platform", "govuk-developers", "govuk-find-and-view", "govuk-publishing", "replatforming"],
+    "locations-api"            = ["govuk-content-interactions-on-platform", "govuk-developers", "govuk-find-and-view", "govuk-platform-security-reliability-team", "replatforming"],
+    "manuals-publisher"        = ["govuk-developers", "govuk-publishing", "govuk-publishing-experience", "replatforming"],
+    "maslow"                   = ["govuk-developers", "govuk-publishing", "govuk-publishing-experience", "replatforming"],
+    "publisher"                = ["govuk-developers", "govuk-publishing", "govuk-publishing-on-platform", "replatforming"],
+    "publishing-api"           = ["govuk-developers", "govuk-publishing", "govuk-publishing-platform", "replatforming"],
+    "release"                  = ["govuk-developers", "govuk-platform-security-reliability-team"],
+    "router"                   = ["govuk-developers", "govuk-find-and-view", "govuk-navigation-presentation", "replatforming"],
+    "router-api"               = ["govuk-developers", "govuk-find-and-view", "govuk-navigation-presentation", "govuk-publishing", "replatforming"],
+    "search-admin"             = ["govuk-developers", "govuk-find-and-view", "replatforming"],
+    "search-analytics"         = ["govuk-developers", "govuk-find-and-view", "replatforming"],
+    "search-api"               = ["govuk-developers", "govuk-find-and-view", "govuk-finding-brexit-team", "replatforming"],
+    "service-manual-publisher" = ["govuk-developers", "govuk-publishing", "govuk-publishing-experience", "replatforming"],
+    "short-url-manager"        = ["govuk-developers", "govuk-publishing", "govuk-publishing-experience", "replatforming"],
+    "signon"                   = ["govuk-access-and-permissions", "govuk-developers", "govuk-publishing", "replatforming"],
+    "smart-answers"            = ["govuk-developers", "govuk-find-and-view", "govuk-frontend", "replatforming"],
+    "specialist-publisher"     = ["govuk-developers", "govuk-publishing", "govuk-publishing-experience", "replatforming"],
+    "static"                   = ["govuk-developers", "govuk-find-and-view", "govuk-frontend", "govuk-navigation-presentation", "replatforming"],
+    "support"                  = ["govuk-developers", "govuk-platform-security-reliability-team", "replatforming"],
+    "support-api"              = ["govuk-developers", "govuk-platform-security-reliability-team", "replatforming"],
+    "transition"               = ["govuk-developers", "govuk-publishing", "govuk-publishing-platform", "replatforming"]
```

Comment:
If we're defining the additional teams in Terraform (and I agree we should!) then we should probably [update this code in govuk-user-reviewer](https://github.com/alphagov/govuk-user-reviewer/blob/c27f2d0b7563aaebad14205712d3174f6bdfa97a/terraform/saas/sentry.tf#L10-L14), because we now _do_ want to overwrite any teams changes done through the UI, making govuk-infrastructure the single source of truth.

Reply:
I've opened a [PR](https://github.com/alphagov/govuk-user-reviewer/pull/978)